pkg/sys/drive_discovery_linux.go

```diff
@@ -222,12 +222,10 @@ func updateFSInfo(device *Device, CDROMs, swaps map[string]struct{}, mountInfos
 
 	majorMinor := fmt.Sprintf("%v:%v", device.Major, device.Minor)
 	if _, found := swaps[majorMinor]; !found {
-		device.MountPoints = mountPointsMap[majorMinor]
-		if len(device.MountPoints) > 0 {
-			device.FirstMountPoint = mountInfos[majorMinor][0].MountPoint
-			device.FirstMountOptions = mountInfos[majorMinor][0].MountOptions
+		if len(device.MountInfos) > 0 {
+			device.FirstMountPoint = device.MountInfos[0].MountPoint
+			device.FirstMountOptions = device.MountInfos[0].MountOptions
```

Comment:
we can remove this as first mountpoint and first mountoptions can be filled from `device.MountInfos`